src/wallet/wallet.h

```diff
@@ -400,6 +400,7 @@ class CWalletTx : public CMerkleTx
     bool IsEquivalentTo(const CWalletTx& tx) const;
 
     bool InMempool() const;
+    uint64_t GetMempoolAncestorCount() const;
```

Comment:
nit: add documentation to this function that the tx must be in the mempool in order to call?